### PR TITLE
Change "dance" to "wiggle"

### PR DESCRIPTION
**Pull request by Amr-Ibra**
“Wiggle” conveys the meaning better than “Dance” for the urgent icon animation.

### CODE REVIEW

File: schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml [hunk 1]
```diff
@@ -280,10 +280,10 @@
       <summary>Isolate volumes, devices and trash windows</summary>
       <description>Consider volume, devices and trash as different application windows and not part of the file manager</description>
     </key>
-    <key type="b" name="dance-urgent-applications">
+    <key type="b" name="wiggle-urgent-applications">
```
Comment (3v1n0) [inline schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml:283]: Please, don't change this, otherwise people's setting won't be changed.

It's fine to have it called differently here.

File: appIcons.js [hunk 2]
```diff
@@ -170,18 +170,18 @@ var DockAbstractAppIcon = GObject.registerClass({
             const icon = this.icon._iconBin;
             this._signalsHandler.removeWithLabel(Labels.URGENT_WINDOWS);
             if (this.urgent) {
-                if (Docking.DockManager.settings.danceUrgentApplications &&
+                if (Docking.DockManager.settings.wiggleUrgentApplications &&
```
Comment (3v1n0) [inline appIcons.js:173]: As per that, revert this change

File: schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml (hunk 1 above)
Comment (Amr-Ibra) [reply schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml]: Reverted.

File: appIcons.js (hunk 2 above)
Comment (Amr-Ibra) [reply appIcons.js]: Reverted.

File: docking.js [hunk 3]
```diff
@@ -572,7 +572,7 @@ const DockedDash = GObject.registerClass({
             Utils.SignalsHandlerFlags.CONNECT_AFTER,
         ], [
             settings,
-            'changed::dance-urgent-applications',
+            'changed::wiggle-urgent-applications',
```
Comment (Amr-Ibra) [inline docking.js:575]: Revert this too? Or not needed?

Comment (3v1n0) [inline docking.js:575]: Revert this too please

File: prefs.js [hunk 4]
```diff
@@ -731,8 +731,8 @@ var Settings = GObject.registerClass({
         updateIsolateLocations();
         isolateLocationsBindings.forEach(s => this._builder.get_object(s).connect(
             'notify::active', () => updateIsolateLocations()));
-        this._settings.bind('dance-urgent-applications',
-            this._builder.get_object('dance_urgent_applications_switch'),
+        this._settings.bind('wiggle-urgent-applications',
```
Comment (3v1n0) [inline prefs.js:734]:
```suggestion
        this._settings.bind('dance-urgent-applications',
```

File: docking.js (hunk 3 above)
Comment (Amr-Ibra) [reply docking.js]: Done

File: prefs.js (hunk 4 above)
Comment (Amr-Ibra) [reply prefs.js]: Done

**Comment by 3v1n0** [comment]
Thanks!
